bleak/backends/bluezdbus/manager.py

```diff
@@ -216,6 +216,28 @@ def _check_device(self, device_path: str) -> None:
         if device_path not in self._properties:
             raise BleakError(f"device '{device_path.split('/')[-1]}' not found")
 
+    def _get_device_property(
+        self, device_path: str, interface: str, property_name: str
+    ) -> Any:
+        self._check_device(device_path)
+        device_properties = self._properties[device_path]
+
+        try:
+            interface_properties = device_properties[interface]
+        except KeyError:
+            raise BleakError(
+                f"Interface {interface} not found for device '{device_path.split('/')[-1]}'"
```

Comment:
Might be helpful to keep the full path for cases where people have multiple Bluetooth adapters.

Reply:
Okay I copied the `_check_device()` message.

Reply:
Ha, OK. I'll leave it up to you.